app/src/test/java/nl/tudelft/b_b_w/model/DatabaseHandlerUnitTest.java

```diff
@@ -42,7 +42,7 @@
     private final String iban = "iban";
     private final String publicKey = "publicKey";
     private Block _block;
-    private final int trustValue = 0;
+    private final int trustValue = TrustValues.INITIALIZED.getValue();
```

Comment:
And calling a get method multiple times when you can call it once and save the address is better for the runtime complexity.